src/dipdup/models/evm_subsquid.py

```diff
@@ -1,13 +1,54 @@
+from enum import Enum
 from typing import Any
 from typing import Generic
+from typing import Literal
 from typing import TypeVar
 
-PayloadT = TypeVar('PayloadT', bound=Any)
+from pydantic import BaseModel
+from pydantic.dataclasses import dataclass
 
+from dipdup.subscriptions import Subscription
 
+PayloadT = TypeVar('PayloadT', bound=BaseModel)
+
+
+# FIXME: Outdated values
+class SubsquidMessageType(Enum):
+    """Enum for filenames in squid archives"""
+
+    blocks = 'blocks.arrow_stream'
+    logs = 'logs.arrow_stream'
+
+
+@dataclass
+class SubsquidEventData:
+    address: str
+    # block_hash: str
+    # block_number: int
+    data: str
+    # index: int
+    # removed: bool
+    topics: tuple[str, ...]
+    # transaction_hash: str
+    # transaction_index: int
```

Comment:
Either hash or id required for merging logs into txs (for operation index) — need to check what index is